Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -82,6 +85,14 @@ class PetSingleDetailsViewController: UIViewController {
     }
 
     private func isPetSaved() {
-        saveSinglePetButton.isEnabled = !singlePetViewModel.isPetSaved()
+        let isEnabled =  !singlePetViewModel.isPetSaved()
+        saveSinglePetButton.isEnabled = isEnabled
```

Comment:
This can be simplified to: 
`saveSinglePetButton.isEnabled = !singlePetViewModel.isPetSaved()`

Reply:
Thank you 👍🏻